cmd/kubeadm/OWNERS

```diff
@@ -12,5 +12,7 @@ reviewers:
 - jamiehannaford
 - kad
 - xiangpengzhao
-- mattmoyer
```

Comment:
Removing Matt is intentional or accidental ?

Reply:
he asked yes